[HW] 1072947

unoblesser opened the pull request:
要解決問題:
由於只能有一筆commit，所以當我們bash的結果錯誤，重新操作完整個流程再次bash，便會出現Your commit number is more than 1
如何解決:
可以到使用者資料夾將整個ncyu-2021-ex1檔案刪除，重新操作完整個流程再次bash即可

Signed-off-by: 1072947 <s1072947@mail.ncyu.edu.tw>

unoblesser commented:
之前PR的內容有誤，已重新PR，麻煩老師再次過目

jrjang commented on README.md:
```diff
@@ -15,6 +15,7 @@ echo "Example 1" > ex1.txt
 [注意] 只能有"一筆"commit
 
 3. 在 ncyu-2021-ex1 資料夾下, 執行以下指令
+[注意] 如果出現 Your commit number is more than 1 ，請刪除 ncyu-2021-ex1 資料夾，重新執行
```
將這一行描述移到"GITHUB_ACCOUNT: github帳號" 的下面. 前面需空一行

jrjang commented:
Hi @unoblesser ,

commit message 可以修改成

當在執行 ex1-test.sh 時，出現以下錯誤訊息，可將 ncyu-2021-ex1 資料夾刪除，重新操作。

Your commit number is more than 1

在 README.md 中，加入相關提示

unoblesser commented:
已依照老師的建議進行修改，麻煩老師再次過目，辛苦您了

jrjang commented on README.md:
```diff
@@ -21,3 +21,5 @@ bash <(curl -s https://raw.githubusercontent.com/jrjang/ncyu-2021/ex1/scripts/ex
 
 GITHUB_ACCOUNT: github帳號
+
+[注意] 如果出現 Your commit number is more than 1 ，請刪除 ncyu-2021-ex1 資料夾>，重新執行
```
"資料夾>，重新執行"，多了一個">"

unoblesser replied:
修改完畢

Commit:
[HW] 1072947

當在執行 ex1-test.sh 時，出現以下錯誤訊息，可將 ncyu-2021-ex1 資料夾刪除，重新操作。

Your commit number is more than 1

在 README.md 中，加入相關提示

Signed-off-by: 1072947 <s1072947@mail.ncyu.edu.tw>

jrjang commented:
Good Job.

unoblesser commented:
謝謝老師耐心的指導
